kubernetes-image/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+ORG_PATH=github.com/c-fs
+REPO_PATH=$ORG_PATH/cfs
+
+go build -a -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"' -o cfs ${REPO_PATH}/server
+docker build -t yunxing/cfs:k8s .
```

Comment:
both docker hub and quay doesn't support organization name "c-fs", so we decide to use some unofficial org name and refine it later. /cc @yunxing